api/src/pcapi/alembic/versions/20240125T155224_04caadcaa652_add_index_to_institution_name.py

```diff
@@ -0,0 +1,33 @@
+"""Add index on EducationalInstitution name
+"""
+
+from alembic import op
+
+from pcapi import settings
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "04caadcaa652"
+down_revision = "6be778edb2f4"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("COMMIT")
+    op.execute("SET SESSION statement_timeout='300s'")
+    op.execute(
+        """
+            CREATE INDEX CONCURRENTLY IF NOT EXISTS
+            "ix_educational_institution_name" ON public.educational_institution USING btree ("name")
```

Comment:
Suggestion : un index sur type + name + city concaténés, et chercher sur cette indexation
```suggestion
            "ix_educational_institution_type_name_city" ON public.educational_institution USING btree ("type" || ' ' || "name" || ' ' || "city" )
```